OpenPNM/Physics/__GenericPhysics__.py

```diff
@@ -63,7 +63,7 @@ def __init__(self, network=None, phase=None, geometry=None,
             self._phases.append(phase)  # Register phase with self
 
         if geometry is not None:
-            if (pores != []) or (throats != []):
+            if (sp.size(pores) > 0) or (sp.size(throats) > 0):
```

Comment:
Let's keep'em, unless pep8 is being a little cry baby about them.  Bool
comparisons can behave very stragely without parens, in my experience.

On Thu, May 14, 2015 at 3:45 PM, Brennan Spellacy <notifications@github.com>
wrote:

> In OpenPNM/Physics/__GenericPhysics__.py
> <https://github.com/PMEAL/OpenPNM/pull/360#discussion_r30355524>:
>
> > @@ -63,7 +63,7 @@ def __init__(self, network=None, phase=None, geometry=None,
> >              self._phases.append(phase)  # Register phase with self
> >
> >          if geometry is not None:
> > -            if (pores != []) or (throats != []):
> > +            if (sp.size(pores) > 0) or (sp.size(throats) > 0):
>
> We don't actually need these parens.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/PMEAL/OpenPNM/pull/360/files#r30355524>.
>